### DISCUSSION TITLE
teal webinar 2023-02

### DISCUSSION THREAD

**Discussion by wobweger** [insightsengineering/teal#799] [Q&A] [answered]
I'm watching https://www.youtube.com/watch?v=nqJsLSLd39A,
Thomas mentioned an upcoming webinar at end of February,
I would very much appreciate being allowed to attend.

**Comment by donyunardi** [comment]
Hi @wobweger, absolutely.
I believe you're referring to the R Consortium webinar in February.

@thomas-neitmann
perhaps you can provide more info about this? (registration, etc.)

**Comment by thomas-neitmann** [comment, marked as answer]
The info is not live on the R Consortium's website just yet but the webinar will take place on February 23rd at 8 am PST.

**Reply by wobweger** [reply]
excellent, merci
looking forward to the webinar.
